Plugins/Flow.Launcher.Plugin.Explorer/Search/SearchManager.cs

```diff
@@ -252,5 +256,16 @@ internal static bool IsEnvironmentVariableSearch(string search)
                    && search != "%%"
                    && !search.Contains('\\');
         }
+
+        // https://stackoverflow.com/a/66877016
+        internal static bool IsSubPathOf(string subPath, string basePath)
```

Comment:
Please move this method to FilesFolders class, which can then be shared by other plugins and flow.